### PR TITLE
Switched array() to []

### PR DESCRIPTION
**Pull request by Arman-Hosseini**
| Q             | A
| ------------- | ---
| Branch?       | 3.4 
| Bug fix?      | no
| New feature?  | no 
| BC breaks?    | no     
| Deprecations? | yes 
| Tests pass?   | no    
| Fixed tickets | #...   
| License       | MIT
| Doc PR        | symfony/symfony-docs#...

### CODE REVIEW

**Comment by derrabus** [comment]
The PHPUnit bridge 3.4 needs to stay compatible with php 5.3. This is why we cannot do the change you’re suggesting.

**Comment by Arman-Hosseini** [comment]
@derrabus Review it again.
Thank you.

File: src/Symfony/Bridge/PhpUnit/Tests/DnsMockTest.php [hunk 1]
```diff
@@ -18,15 +18,15 @@ class DnsMockTest extends TestCase
 {
     protected function tearDown()
     {
-        DnsMock::withMockedHosts(array());
+        DnsMock::withMockedHosts([]);
```
Comment (derrabus) [inline src/Symfony/Bridge/PhpUnit/Tests/DnsMockTest.php]: PhpUnitBridge has to remain compatible with php 5.3. This change would break that compatibility.

**Review by derrabus — CHANGES_REQUESTED** [review]

**Comment by lyrixx** [comment]
Closing as explained by @derrabus

**Pull request closed without merging**